src/modDesc.xml

```diff
@@ -30,19 +34,23 @@ Changelog ${version}:
 		</en>
 		<de>
 			<![CDATA[
-Dieser Mod hilft das Vertragssystem zu verbessern, sowohl im Einzelspieler und Multiplayer.
+"Mach ich jetzt diese drei Düngemissionen, oder doch lieber den 2.3 ha Kartoffelernte-Vertrag?" "Wieviel Flüssigdünger brauche ich für diesen Auftrag?" Falls Sie sich je derartige Fragen gestellt haben, hilft Ihnen dieser Mod bei den Antworten. Er verbessert das Vertragsmanagement, sowohl im Einzelspieler- als im Multiplayer-Modus.
 
-- Die maximale Anzahl und die Anzahl der generierten Missionen werden jetzt automatisch an die Anzahl der Felder in der Karte angepasst.
+- Die Maximalzahl sowie die Anzahl der jeweils neu generierten Missionen werden automatisch an die Zahl der Felder auf der Karte angepasst.
  
-- Sie können jetzt über die Schaltfläche "Neue Verträge" neue Missionen generieren und diese sogar alle auf dem Vertragsverwaltungsbildschirm löschen.
+- Sie können über die Schaltfläche "Neue Verträge" sofort neue Missionen generieren oder alle mit dem Button "Liste löschen" entfernen.
 
-- Verträge werden jetzt nach Typ und Feldnummer sortiert, um das Auffinden des gewünschten Vertrags zu erleichtern.
+- Mit dem "Details" Button kann die Anzeige der zusätzlichen Vertragsdetails ein- und ausgeschaltet werden.
+
+- Sie können die Vertragsliste sortieren nach Vertragstyp/ Feldnummer, nach Gesamtprofit, und nach Profit pro Minute, um einen gewünschten Vertrag schnell zu finden.
  
 - Sie können mehr als einen Vertrag gleichzeitig aktivieren.
  
-- Die Feldnummernanzeige (auf der Minikarte) der Felder, auf denen Sie Verträge aktivieren, beginnt zu blinken, um Ihnen den richtigen Weg zu zeigen.
+- Die Feldnummernanzeigen (auf der Minikarte) der Felder mit Ihren aktiven Verträgen blinken, um Ihnen den Weg zu zeigen.
 
 Changelog ${version}:
+- Zusätzliche Details: Neben Feldnummer, Feldgröße, Auftragsart werden weitere Details des Auftrags angezeigt. Z.B. die geschätzten Kosten des Verbrauchsmaterials (Dünger, Pflanzenschutzmittel, Saatgut). Bei Ernte- und Gras-Missionen wird die abzuliefernde Mindestmenge angezeigt, sowie die Erntemenge, die Sie behalten (und verkaufen) können. Daraus ermittelt der Mod den Gesamtertrag der Mission, also Belohnung minus Kosten für Dünge-, Spritz-, und Sähen-Verträge; und Belohnung plus Wert der behaltenen Erntemenge bei den Ernte- und Gras-Verträgen. Er schätzt sogar die Gesamtzeit, die Sie für die Erledigung des Auftrags benötigen werden, und berücksichtigt dafür Arbeitsgeschwindigkeit und Arbeitsbreite der entsprechenden Leasing-Geräte, die für den Vertrag angeboten werden.
+
 - Konfliktverhütung mit anderen Mods hinzugefügt
 - Sortierung der Vertragslisten hinzugefügt
 - Es wurden viele neue Fahrzeugkombinationen für Verträge hinzugefügt
```

Comment:
updated only English/ German. Other languages need to be done.

Reply:
I will do the french translation when this will be merged into the main project.